internal/keypairs/keypair.go

```diff
@@ -103,7 +103,7 @@ func SaveEncodedKeypair(cfg Configurable, secretsClient *secretsapi.Client, encK
 		PublicKey:           &encKeypair.EncodedPublicKey,
 	})
 
-	if _, err := secretsClient.Keys.SaveKeypair(params, authentication.New(cfg).ClientAuth()); err != nil {
+	if _, err := secretsClient.Keys.SaveKeypair(params, authentication.LegacyGet().ClientAuth()); err != nil {
```

Comment:
I presume you have a good reason to use LegacyGet(), so I'll let it slide this time.

Reply:
Yeah; secrets code is itself legacy and refactoring this out for this bug would be far too costly.